core/src/widget_tree.rs

```diff
@@ -28,8 +28,8 @@ pub(crate) struct WidgetTree {
 impl WidgetTree {
   pub fn init(&mut self, wnd: Weak<Window>) { self.wnd = wnd; }
 
-  pub fn set_root(&mut self, root: WidgetId) {
-    self.root = root;
+  pub fn set_content(&mut self, content: WidgetId) {
+    self.root.append(content, &mut self.arena);
```

Comment:
What happen if we call `set_content` twice with different widget?

Reply:
yeah, here should clean the old content